main.tf

```diff
@@ -26,7 +26,7 @@ module "aws_vpc" {
 module "aws_elb" {
   enable = "${var.vpc_enable_bastion_host}"
 
-  source = "github.com/Flaconi/terraform-aws-elb?ref=v0.1.5"
+  source = "github.com/Flaconi/terraform-aws-elb?ref=v0.1.8"
```

Comment:
@danvaida 
I suppose this should be `v0.1.9` as `v0.1.8` is already released and you're referring to the unreleaesd PR: https://github.com/Flaconi/terraform-aws-elb/pull/5

Reply:
Clarified [here](https://github.com/Flaconi/terraform-aws-vpc/pull/7/files#r544979801).